### PR TITLE
Update Ambassador to version 1.12.1

### PR DESCRIPTION
**Pull request by rhs**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ numbering uses [semantic versioning](http://semver.org).
 
 ## Next Release
 
+## v6.5.22
```
Comment (khussey) [inline CHANGELOG.md:8]: Needs to be 6.6.0.

File: Chart.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: 1.12.0
-ossVersion: 1.12.0
+appVersion: 1.12.1
+ossVersion: 1.12.1
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 6.5.21
```
Comment (khussey) [inline Chart.yaml:6]: 6.6.0

**Review by khussey — CHANGES_REQUESTED** [review]
The Helm chart release for 1.12.1 needs to be 6.6.0.

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +24,8 @@ numbering uses [semantic versioning](http://semver.org).
 ## v6.5.20
 
 - Update Ambassador to version v1.11.2: [CHANGELOG](https://github.com/datawire/ambassador/blob/master/CHANGELOG.md)
+- Bugfix: The endpoint routing resolver will now properly watch services with mappings that define the service field with an explicit port.
```
Comment (khussey) [inline CHANGELOG.md:27]: Why are these entries here and not above under v6.6.0?

**Review by khussey — CHANGES_REQUESTED** [review]

**Review by acookin — APPROVED** [review]

**Pull request merged**